<launch>

    <arg name="camera" value="/D435_head_camera/color" />
    <arg name="image" value="image_raw" />
    <arg name="dictionary" value="0" />
    <arg name="fiducial_len" value="0.15" />
    <arg name="do_pose_estimation" value="true" />
    <arg name="verbose" value="false" />

    <include file="$(find aruco_detect)/launch/aruco_detect.launch" pass_all_args="true"/>

</launch>